From 48b860b69f841ece6effd3f01ca39e93518e35ac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 10 Sep 2019 16:02:44 +0200
Subject: [PATCH 040/152] ci: Turn on -Werror

Now that all compiler warnings are fixed, let's try to keep it that
way by making warnings fatal during CI.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/709
---
 .gitlab-ci.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index cedd587c5..03c7f9786 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -54,7 +54,7 @@ build:
         - meson mutter mutter/build --prefix=/usr -Dtests=false
         - ninja -C mutter/build install
     script:
-        - meson . build -Dbuiltype=debugoptimized -Dman=false
+        - meson . build -Dbuiltype=debugoptimized -Dman=false --werror
         - ninja -C build
         - ninja -C build install
     <<: *only_default
-- 
2.23.0

